packages/showcases/css/stories/components/button/examples/conversion.html

```diff
@@ -38,7 +47,16 @@
   </button>
 </div>
 
-<div class="block">
+<div
+  class="block"
+  style="
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    width: 60%;
+  "
+>
```

Comment:
I don't really like this effect. Is it possible instead to use margins between buttons inside the showcase?
Thanks.

Reply:
fixed